app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/template.rb

```diff
@@ -1,2 +1,28 @@
+# frozen_string_literal: true
+
+# Provide CloudManager support for IBM CLoud VPC templates.
 class ManageIQ::Providers::IbmCloud::VPC::CloudManager::Template < ManageIQ::Providers::CloudManager::Template
+  include_concern 'ManageIQ::Providers::IbmCloud::VPC::CloudManager::VmOrTemplateShared'
+
+  supports :provisioning do
+    if ext_management_system
+      unsupported_reason_add(:provisioning, ext_management_system.unsupported_reason(:provisioning)) unless ext_management_system.supports_provisioning?
+    else
+      unsupported_reason_add(:provisioning, _('not connected to ems'))
+    end
+  end
+
+  # Get a new CloudTool VPC object.
+  # @param connection [ManageIQ::Providers::IbmCloud::CloudTools::Vpc] An existing connection.
+  # @return [ManageIQ::Providers::IbmCloud::CloudTools::Vpc] The passed in connection or a new one.
+  def provider_object(connection = nil)
+    connection || ext_management_system.connect
```

Comment:
Yeah see this isn't what `provider_object` should be doing, if you want the ems connection you can just use `with_provider_connection`.

If you do want the template object you'll have to do the equivalent of:
```
  # Used in with_provider_object to scope SDK to this instance.
  def provider_object(vpc)
    vpc.instances.instance(ems_ref)
  end
```

Reply:
Another copy/paste from Power